ui-components/src/locales/es.json

```diff
@@ -384,7 +384,16 @@
   "listingFilters.program.Veterans": "Veteranos",
   "listingFilters.clear": "Borrar",
   "listingFilters.section8": "Acepta vales de elección de vivienda de la Sección 8",
-  "listingFilters.buttonTitleExtended": "Encuentre ofertas para usted",
+  "listingFilters.buttonTitleExtended": "Encuentre alquileres para usted",
```

Comment:
There is no gap between the buttons when in Spanish since the text is too large to be in one row. 
![image](https://user-images.githubusercontent.com/42942267/197554009-9a605ab5-3122-445e-8bd4-264c62a4f20a.png)

A similar issue happens in Arabic. The buttons touch. 
![image](https://user-images.githubusercontent.com/42942267/197554206-872d962f-5d99-4d10-b0c6-6575bdafa518.png)

Reply:
@ludtkemorgan This should be addressed now!